tools/bin/goreleaser_utils

```diff
@@ -6,12 +6,11 @@ set -xe
 before_hook() {
   local -r lib_path=$PWD/tmp
 
-  mkdir -p "$lib_path/libs"
-  # Copy over all platform versions of the wasmvm library
-  cp -f "$(go list -json -m github.com/CosmWasm/wasmvm | jq -r '.Dir')"/internal/api/libwasmvm.* "$lib_path/libs"
-
   install_local_plugins
   install_remote_plugins
+  mkdir -p "$lib_path/libs"
+  # Copy over all platform versions of the wasmvm library
+  cp -f /go/pkg/mod/github.com/\!cosm\!wasm/wasmvm@v*/internal/api/libwasmvm.* "$lib_path/libs"
```

Comment:
This hard coded path doesn't work. We can't `go list` b/c it is no longer imported. The makefile uses `go mod download` now, but invoking that again would require duplicating the hash. Perhaps `go env GOMODCACHE` could be leveraged?

Reply:
This seems to work